src/util/segmentation/boundaries/getBoundingBoxAroundCircle.js

```diff
@@ -0,0 +1,24 @@
+import getCircleCoords from '../../getCircleCoords';
+
+export default function getBoundingBoxAroundCircle(evt) {
+  const { handles } = evt.detail;
+  const { width, height } = evt.detail.image;
+  const circleCoordinates = getCircleCoords(handles.start, handles.end);
+
+  let xMax = circleCoordinates.width + circleCoordinates.left;
+  let xMin = circleCoordinates.left;
+  let yMax = circleCoordinates.top + circleCoordinates.height;
+  let yMin = circleCoordinates.top;
+
+  xMin = Math.round(xMin);
+  yMin = Math.round(yMin);
+  xMax = Math.round(xMax);
+  yMax = Math.round(yMax);
```

Comment:
Is it safe to round these? For the tool we use them for, it feels fine, but I don't know if I like the result for other applications. We had an issue with `getCircle` recently in the v3 brush implementation that was caused by some unexpected rounding behavior (IIRC)

Reply:
We have to round them to _something_, as they are indicies.

Reply:
Math.floored.